lib/aio/jobcontext.py

```diff
@@ -105,19 +106,33 @@ async def __aenter__(self) -> Self:
                     }
                 self.default_image = get_str(container, 'default-image')
 
-            with get_nested(self.config, 'logs') as logs:
-                driver = get_str(logs, 'driver')
+            with get_nested(self.config, "logs") as logs:
+                driver = get_str(logs, "driver")
```

Comment:
Not that it matters, but I'm curiuos: You have been very consistently converted our code from double to single quotes. Why the sudden change?

Reply:
This was accidental.  The main story is that I have ` f` on muscle memory from working with Rust (where absolutely everyone is using `rust-fmt`) and `test.thing` (where I use `ruff format`).  I keep accidentally pressing ` f` to fix line wrapping when working on other projects but of course this accidentally reformats the whole file.  I clearly missed backing this change out.